src/lightning_app/utilities/cli_helpers.py

```diff
@@ -281,7 +281,7 @@ def _check_version_and_upgrade():
         prompt = f"A newer version of {__package_name__} is available ({new_version}). Would you like to upgrade?"
 
         if click.confirm(prompt, default=True):
-            command = f"pip install '{__package_name__}=={new_version}'"
+            command = f"pip install {__package_name__}=={new_version}"
```

Comment:
```suggestion
            command = f'pip install "{__package_name__}=={new_version}"'
```

Reply:
Double quotes and single quotes are not treated equally in shells. Using single quotes inside is probably safer across shells